extern/CMakeProject-mad.cmake

```diff
@@ -1,36 +1,47 @@
-list(APPEND MAD_SRC
-  "mad-0.15.1b/bit.c"
-  "mad-0.15.1b/decoder.c"
-  "mad-0.15.1b/fixed.c"
-  "mad-0.15.1b/frame.c"
-  "mad-0.15.1b/huffman.c"
-  "mad-0.15.1b/layer12.c"
-  "mad-0.15.1b/layer3.c"
-  "mad-0.15.1b/stream.c"
-  "mad-0.15.1b/synth.c"
-  "mad-0.15.1b/timer.c"
-  "mad-0.15.1b/version.c"
+set(MAD_DIR "libmad-0.15.1b")
+
+set(MAD_SRC
+  "${MAD_DIR}/bit.c"
+  "${MAD_DIR}/decoder.c"
+  "${MAD_DIR}/fixed.c"
+  "${MAD_DIR}/frame.c"
+  "${MAD_DIR}/huffman.c"
+  "${MAD_DIR}/layer12.c"
+  "${MAD_DIR}/layer3.c"
+  "${MAD_DIR}/stream.c"
+  "${MAD_DIR}/synth.c"
+  "${MAD_DIR}/timer.c"
+  "${MAD_DIR}/version.c"
+)
+
+set(MAD_HPP
+  "${MAD_DIR}/bit.h"
+  "${MAD_DIR}/decoder.h"
+  "${MAD_DIR}/fixed.h"
+  "${MAD_DIR}/frame.h"
+  "${MAD_DIR}/global.h"
+  "${MAD_DIR}/huffman.h"
+  "${MAD_DIR}/layer12.h"
+  "${MAD_DIR}/layer3.h"
+  "${MAD_DIR}/stream.h"
+  "${MAD_DIR}/synth.h"
+  "${MAD_DIR}/timer.h"
+  "${MAD_DIR}/version.h"
 )
 
-list(APPEND MAD_HPP
-  "mad-0.15.1b/bit.h"
-  "mad-0.15.1b/decoder.h"
-  "mad-0.15.1b/fixed.h"
-  "mad-0.15.1b/frame.h"
-  "mad-0.15.1b/global.h"
-  "mad-0.15.1b/huffman.h"
-  "mad-0.15.1b/layer12.h"
-  "mad-0.15.1b/layer3.h"
-  "mad-0.15.1b/mad.h"
-  "mad-0.15.1b/stream.h"
-  "mad-0.15.1b/synth.h"
-  "mad-0.15.1b/timer.h"
-  "mad-0.15.1b/version.h"
+set(MAD_DAT
+  "${MAD_DIR}/D.dat"
+  "${MAD_DIR}/imdct_s.dat"
+  "${MAD_DIR}/qc_table.dat"
+  "${MAD_DIR}/rq_table.dat"
+  "${MAD_DIR}/sf_table.dat"
 )
 
-source_group("" FILES ${MAD_SRC} ${MAD_HPP})
+source_group("Source Files" FILES ${MAD_SRC})
+source_group("Header Files" FILES ${MAD_HPP})
+source_group("Data Files" FILES ${MAD_DAT})
```

Comment:
Why include the data files? Granted, that's almost like asking why include the header files, but I'm curious.